cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -34,7 +35,17 @@ If you want to create a cluster in an unavailable region, please [contact Suppor
 You do not need an account with the cloud provider you choose in order to create a cluster on that cloud provider. The cluster is created on infrastructure managed by Cockroach Labs. If you have existing cloud services on either GCP or AWS that you intend to use with your {{ site.data.products.serverless }} cluster, you should select that cloud provider and the region closest to your existing cloud services to maximize performance.
 {{site.data.alerts.end}}
 
-## Step 3. Set resource limits
+## Step 3. Select the region(s)
+
+In the **Regions** section, select a region. For optimal performance, select the cloud provider and region nearest to where your SQL clients, applications, or external data are located. For example, if your client application is deployed in GCP's `us-east1` region, select GCP as your deployment environment and select `us-east1` as your cluster's region. If you want to create a cluster in an unavailable region, [contact Support](https://support.cockroachlabs.com) or your Cockroach Labs account team.
```

Comment:
This should be ok for now. Perhaps over time we can make this more oriented toward multi-region, moving away from the original single-region orientation.